cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -293,3 +293,35 @@ def test_provisioning_schedule(provisioner, provider, prov_data, vm_name):
     template_name = provider.data['provisioning']['template']
 
     provisioner(template_name, prov_data, delayed=provision_time)
+
+
+@pytest.mark.rhv2
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(RHEVMProvider),
```

Comment:
Instead of uncollect on type, you should use the provider mark with `override` kwarg

```
@pytest.mark.provider([RHEVMProvider],
                                      required_fields=[['provisioning', 'template'],
                                                                 ['provisioning', 'host'],
                                                                 ['provisioning', 'datastore'],
                                       override=True)
```

Reply:
Thanks, I was wondering if there is such a way to do this. Done.